bin/resty

```diff
@@ -529,6 +530,7 @@ resty [options] [lua-file [args]]
 Options:
     -c NUM              Set maximal connection count (default: 64).
     -e PROG             Run the inlined Lua code in "prog".
+    -l LIB              Require library "lib".
```

Comment:
I think we sort the options alphabetically. Please put this line to the right place. Thanks!

Reply:
I tried to keep it the same as lua/luajit interpreter. There `-l` shows right after `-e`. That makes sense to me, because they do quite similar thing.

But if you wish it on the bottom, so be it.

Reply:
@mikz Alphabetically sorted options in the usage output is much easier for the user to find when we add more and more options. We do not try to be 100% compatible with the `luajit` command-line utility's behavior anyway.